docs/REST API/rest-api.md

```diff
@@ -7035,6 +7063,7 @@ Either `query` or `query_id` must be provided.
 | platform    | string  | body | Comma-separated target platforms, currently supported values are "windows", "linux", "darwin". The default, an empty string means target all platforms. |
 | critical    | boolean | body | _Available in Fleet Premium_. Mark policy as critical/high impact. |
 | calendar_events_enabled    | boolean | body | _Available in Fleet Premium_. Whether to trigger calendar events when policy is failing. |
+| software_title_id  | integer | body | _Available in Fleet Premium_. ID of software title to install if the policy fails. |
```

Comment:
@jacobshandling How do we unset this? (`"software_title_id": null`?)

Reply:
OK, then:
- `"software_title_id"` not set in the PATCH request, won't change the value.
- `"software_title_id": null` will unset the software title from the policy.
- `"software_title_id": X` will set the value to the given integer.

Reply:
Sounds good!

Reply:
@lucasmrod @jacobshandling What do we do in other PATCH endpoints when we want to unset some option? Do we use `null`?

Reply:
When it's a string it's just about using the empty string, maybe this is the first time we have the need to unset a integer on a PATCH request? Cannot find an example.

Another alternative is: if `"software_title_id"` is NOT set in the `PATCH` request then the backend will unset it from the policy. But this one is less explicit, and if you don't want to change the value of `"software_title_id"` for a policy on a PATCH request you'd have to send it with the current value (different than the other fields where if you don't set a field on the PATCH then you don't want the backend to modify it).

Reply:
@lucasmrod I think it makes sense to be explicit and specify `null` if you want to unset value.

Reply:
>     "software_title_id" not set in the PATCH request, won't change the value.
>     "software_title_id": null will unset the software title from the policy.
>     "software_title_id": X will set the value to the given integer.

@lucasmrod fyi the first case should never happen, since the UI won't make a call for unchanged policies